transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollDatagramChannel.java

```diff
@@ -711,7 +703,7 @@ private boolean recvmsg(EpollRecvByteAllocatorHandle allocHandle,
                 return false;
             }
             byteBuf.writerIndex(bytesReceived);
-            InetSocketAddress local = localAddress();
+            InetSocketAddress local = (InetSocketAddress) localAddress();
             DatagramPacket packet = msg.newDatagramPacket(byteBuf, local);
```

Comment:
I think we can get away with making `newDatagramPacket` take a `SocketAddress` recipient, instead of doing this cast.